OrbitGl/CaptureDeserializer.cpp

```diff
@@ -39,11 +54,11 @@ ErrorMessageOr<void> CaptureDeserializer::Load(const std::string& filename) {
     return ErrorMessage("Error opening the file for reading");
   }
 
-  return Load(file);
+  return Load(file, time_graph);
 }
 
-bool CaptureDeserializer::ReadMessage(google::protobuf::Message* message,
-                                      google::protobuf::io::CodedInputStream* input) {
+bool ReadMessage(google::protobuf::Message* message,
```

Comment:
Can this method be also moved to an internal namespace?

Reply:
It is called in _CaptureDeserializerFuzzerInfo_ so I didn't move it inside the internal because of that. Do you think I should anyway and make a call to capture_deserializer::internal::Load instead in there?

Reply:
Discussed offline. I misunderstood and thought the comment was referring to _Load_ instead of _ReadMessage_. _ReadMessage_ method can, indeed, be moved to internal so I have done so. Thanks!